lib/console.js

```diff
@@ -122,9 +122,10 @@ Console.prototype.timeEnd = function timeEnd(label) {
     return;
   }
   const duration = process.hrtime(time);
-  const ms = duration[0] * 1000 + duration[1] / 1e6;
-  this.log('%s: %sms', label, ms.toFixed(3));
+  const ms = (duration[0] * 1000 + duration[1] / 1e6).toFixed(3);
```

Comment:
I think it would be more useful to return the elapsed time as a number rather than a truncated string.